docs/changes.rst

```diff
@@ -26,6 +26,8 @@ v0.4
   section/New section": "content"})`` to add "content" a new subsection called
   "New section" to an existing section called "Existing section". :pr:`203` by
   `Benjamin Bossan`_.
+- Support more array-like data types for tabular data and list-like data types
```

Comment:
I didn't notice earlier, but since we released v0.4 in December, this should be added to a new section "v0.5".